scripts/run_cc_vm_checks.sh

```diff
@@ -3,9 +3,6 @@
 # set -x
 set -e
 
-echo "Verifying CycleCloud process"
-ps -ef | grep jsvc
```

Comment:
If this is failing then there's really something wrong with the image...   

Generally having to change a test is a sign of a regression unless there was an intentional change that changes the test requirement.
We should leave this line in for now.